cypress/support/validation.js

```diff
@@ -0,0 +1,37 @@
+const checkInlineAndSummaryErrorMessage = (expectedMessage) => {
+    cy.get('.govuk-error-summary__list').contains(expectedMessage);
```

Comment:
<img width="887" alt="Screenshot 2022-04-01 at 11 37 52" src="https://user-images.githubusercontent.com/58443816/161247678-6fe49e82-ca68-4fcb-b4fb-7ee837502043.png">

Reply:
can generically target summary and inline error messages. assuming they are the same as per GDS.